rllib/env/wrappers/pettingzoo_env.py

```diff
@@ -82,58 +84,22 @@ def __init__(self, env):
         self.env = env
         env.reset()
 
-        # Since all agents have the same spaces, do not provide full observation-
-        # and action-spaces as Dicts, mapping agent IDs to the individual
-        # agents' spaces. Instead, `self.[action|observation]_space` are the single
-        # agent spaces.
-        self._obs_space_in_preferred_format = False
-        self._action_space_in_preferred_format = False
-
-        # Collect the individual agents' spaces (they should all be the same):
-        first_obs_space = self.env.observation_space(self.env.agents[0])
-        first_action_space = self.env.action_space(self.env.agents[0])
-
-        for agent in self.env.agents:
-            if self.env.observation_space(agent) != first_obs_space:
-                raise ValueError(
-                    "Observation spaces for all agents must be identical. Perhaps "
-                    "SuperSuit's pad_observations wrapper can help (useage: "
-                    "`supersuit.aec_wrappers.pad_observations(env)`"
-                )
-            if self.env.action_space(agent) != first_action_space:
-                raise ValueError(
-                    "Action spaces for all agents must be identical. Perhaps "
-                    "SuperSuit's pad_action_space wrapper can help (usage: "
-                    "`supersuit.aec_wrappers.pad_action_space(env)`"
-                )
-
-        # Convert from gym to gymnasium, if necessary.
-        self.observation_space = convert_old_gym_space_to_gymnasium_space(
-            first_obs_space
-        )
-        self.action_space = convert_old_gym_space_to_gymnasium_space(first_action_space)
-
         self._agent_ids = self.env.agents
```

Comment:
`self._agent_ids` should be a set, so this line should be changed to:

```python
self._agent_ids = set(self.env.agents)
```

Reply:
done